kasa/smart/smartchilddevice.py

```diff
@@ -21,6 +21,17 @@ class SmartChildDevice(SmartDevice):
     This wraps the protocol communications and sets internal data for the child.
     """
 
+    CHILD_DEVICE_TYPE_MAP = {
```

Comment:
This refactoring should go into its own PR to keep this purely (or at least mostly) related to the dump_devinfo & smartcamera.

Reply:
Done in https://github.com/python-kasa/python-kasa/pull/1173